tests/cli-exact.rs

```diff
@@ -1,7 +1,9 @@
 //! Yet more cli test cases. These are testing that the output
 //! is exactly as expected.
 
-use rustup_mock::clitools::{
+pub mod mock;
```

Comment:
Does this approach mean we end up spending CPU compiling mock over and over?

I'm not entirely upset if the answer is yes, but if so, could it be made a `#[cfg(test)] pub mod mock` in src/ instead?  Would that work?

Reply:
Yeah we do.  In cargo this is fixed by making one "testsuite" binary, with all tests.

I'll try the cfg test approach.

Reply:
Doesn't work.